Apps/WebClient/src/ClientApp/app/store/modules/auth/mutations.ts

```diff
@@ -10,6 +11,9 @@ export const mutations: MutationTree<AuthState> = {
     Vue.set(state.authentication, "scopes", user.scopes);
     Vue.set(state.authentication, "idToken", user.id_token);
     Vue.set(state.authentication, "user", user.profile);
+    axios.defaults.headers.common = {
```

Comment:
This looks very odd. Why do we need the Bearer token at the mutation level? If the state needs lets pass it as a parameter instead. Also, lets not set anything that is not state related at this point